collections/_article/release-candidate-godot-4-2-rc-2.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Release candidate: Godot 4.2 RC 2"
+excerpt: "The first release candidate brought another crop of regression reports, which we've now fixed!"
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/release-candidate-godot-4-2-rc-2.webp
+image_caption_title: "2D platformer prototype"
+image_caption_description: "An open source game by Securas"
+date: 2023-11-24 17:00:00
+---
+
+Since our first release candidate for Godot 4.2 a week ago, a number of regressions have been reported by new testers who were starting to test using Godot 4.2 in production. So we've been busy this week triaging, testing, debugging, and fixing regressions to reach a point where we're confident that we're ready for the stable release.
```

Comment:
```suggestion
Since our [first release candidate for Godot 4.2](/article/release-candidate-godot-4-2-rc-1/) a week ago, a number of regressions have been reported by people who were starting to test using Godot 4.2 in production. So we've been busy this week triaging, testing, debugging, and fixing regressions to reach a point where we're confident that we're ready for the stable release.
```
Please double-check I have the correct link syntax! I changed "testers" to avoid having "testers" and "test" in close proximity. Could alternatively say "reported by new testers who were starting to use Godot 4.2 in production"?

Reply:
I like the latter! Edited your suggestion to commit it.